### PR TITLE
Redesigned code view options in lb

### PR DESCRIPTION
**Pull request by MeyAyre**
There were a lot of confusing UI things with the code view options in levelbuilder that needed to be improved.

Changes made:

- Changed the name of the header from "Text mode (showing code) at start" to "Code View When Level Starts"

- Aligned radio buttons with labels 

    - This is going to be used for a future PR

**Before**:
image

**After:** 
image

### CODE REVIEW

File: dashboard/app/views/levels/editors/_droplet.html.haml [hunk 1]
```diff
@@ -21,13 +21,17 @@
...
-  = f.radio_button :text_mode_at_start, 'false'
-  = f.label :text_mode_at_start, 'Start with user preferred setting', :value => 'false'
-  = f.radio_button :text_mode_at_start, 'blocks'
-  = f.label :text_mode_at_start, 'Always start with blocks', :value => 'blocks'
-  = f.radio_button :text_mode_at_start, 'true'
-  = f.label :text_mode_at_start, 'Always start with text', :value => 'true'
+  = f.label :text_mode_at_start, 'Code View When Level Starts'
+  %p What mode the code is presented as when the level loads
+  .alignment-box
+    = f.radio_button :text_mode_at_start, 'false', :checked => true
```
Comment (Hamms) [inline dashboard/app/views/levels/editors/_droplet.html.haml]: just double-checking: this just means the form will _default_ to this value, but will upon future loads be set to whatever it was changed to, right?

File: apps/style/code-studio/levelbuilder.scss [hunk 2]
```diff
@@ -178,3 +178,13 @@ table.checkboxes {
     background-position: -313px -119px;
   }
 }
+
+.alignment-box{
+  padding-top: 1%;
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-start;
+  label{
```
Comment (Hamms) [inline apps/style/code-studio/levelbuilder.scss]: nit: should be a space between the selector and bracket here and above

File: apps/style/code-studio/levelbuilder.scss [hunk 3]
```diff
@@ -178,3 +178,13 @@ table.checkboxes {
     background-position: -313px -119px;
   }
 }
+
+.alignment-box{
```
Comment (Hamms) [inline apps/style/code-studio/levelbuilder.scss]: does `box` in this content refer to something specific? If not, could this perhaps be named something that uses existing terms like `aligned-input-group`?

File: dashboard/app/views/levels/editors/_droplet.html.haml (hunk 1 above)
Comment (MeyAyre) [reply dashboard/app/views/levels/editors/_droplet.html.haml]: Yes

File: apps/style/code-studio/levelbuilder.scss (hunk 3 above)
Comment (MeyAyre) [reply apps/style/code-studio/levelbuilder.scss]: Sounds good to me

**Pull request merged**